Subtester/ProjectSettings/ProjectSettings.asset

```diff
@@ -160,9 +162,9 @@ PlayerSettings:
     iPhone: 0
     tvOS: 0
   overrideDefaultApplicationIdentifier: 0
-  AndroidBundleVersionCode: 1
-  AndroidMinSdkVersion: 22
-  AndroidTargetSdkVersion: 0
+  AndroidBundleVersionCode: 97
+  AndroidMinSdkVersion: 29
```

Comment:
That's a pretty high minSdkVersion... Did you have issues with a lower one?

Reply:
I can try lower ones. I suppose this would affect us if devs are using test devices with lower sdk versions, right?

Reply:
Yeah basically... Also, to make sure we keep the SDK compatible with lower versions of android as well.